IFC4x3/PropertyConstants/c/COPPER_0UGa0Mbq5BGuUCD4GIJzff.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="COPPER_0UGa0Mbq5BGuUCD4GIJzff" Name="COPPER" UniqueId="1e424016-9741-4b43-878c-3444124fda69">
```

Comment:
Would this not be `Material Single` on `IfcCableSegment`?

Reply:
not sure i would go as far as defining a material definition for a copper cable, as that is not the only material in the cable... this is more of a cable type and a connection type for the product than a material definition. @czapplitec ??

Reply:
I was also thinking of using material at the very begining (back in 2019), but this is really a domain view of cables for data communication. There are just two high-level types of data cables, either copper or fiber, based on these two types, there are two different ways to build a communication system, so the junction box used for these two types are different. The meaning of these two types are more than just material.